packages/ampath-esm-clinic-dashboard-hiv-lists/src/custom-hooks/useBaseEtlUrl.tsx

```diff
@@ -0,0 +1,3 @@
+const ETL_SERVER_KEY = "appSettings.etlServer";
+
+export const useBaseEtlUrl = () => window.localStorage.getItem(ETL_SERVER_KEY);
```

Comment:
Maybe we should provide a default value in the unlikely case where that key doesn't exist to avoid a `null` value? Dunno.

`export const useBaseEtlUrl = (): string => window.localStorage.getItem(ETL_SERVER_KEY) || 'some default value';`